drone_optimal_control/DroneExtendedState state
rocket_utils/DroneGimbalControl gimbal_control
std_msgs/Header header
